doc-include/migrate-mac.dtx

```diff
@@ -358,14 +358,14 @@
 %   \item \cs{unvxh} has been replaced with \cs{Xunvxh} and \cs{unvxhX} with two mandatory arguments.
 % \end{itemize}
 % \subsection{Migration to \reledmac~2.1.0}
-% \Reledmac~2.1.0 fix some bug when using \cs{Xbhooknote} and \cs{bhooknoteX} not in order to execute code at the beginning of each notes, but to insert content of at the beginning of each notes.
+% \Reledmac~2.1.0 fix some bugs when using \cs{Xbhooknote} and \cs{bhooknoteX}, in order to insert content of at the beginning of each note. This is not intended to execute code at the beginning of each note.
```

Comment:
I did say almost exactly what you said, but using the clause “not in order to …” first will make many readers feel like they have lost the purpose of the sentence. Putting the negative information afterwards and in a separate sentence actually “pumps up” the negative use even more.

I admit that some such things are matters of taste, but you should know that I do quite a bit of writing and public speaking. In the end, of course, it’s your decision and I won’t be offended by your decisions in any way! I promise!   :-)

Mark


On Sep 4, 2015, at 16:20 , Maïeul Rouquette <notifications@github.com> wrote:

> In doc-include/migrate-mac.dtx:
> 
> > @@ -358,14 +358,14 @@
> >  %   \item \cs{unvxh} has been replaced with \cs{Xunvxh} and \cs{unvxhX} with two mandatory arguments.
> >  % \end{itemize}
> >  % \subsection{Migration to \reledmac~2.1.0}
> > -% \Reledmac~2.1.0 fix some bug when using \cs{Xbhooknote} and \cs{bhooknoteX} not in order to execute code at the beginning of each notes, but to insert content of at the beginning of each notes.
> > +% \Reledmac~2.1.0 fix some bugs when using \cs{Xbhooknote} and \cs{bhooknoteX}, in order to insert content of at the beginning of each note. This is not intended to execute code at the beginning of each note.
> @bishopcranmer why do you change the position. You say exactly the opposition as I say. I would like to say people should not use bhooknote to insert content, but should use it to execute code...
> 
> —
> Reply to this email directly or view it on GitHub.
>